pkg/sippyserver/metrics/metrics.go

```diff
@@ -18,7 +18,6 @@ import (
 	"github.com/openshift/sippy/pkg/apis/cache"
 	v1 "github.com/openshift/sippy/pkg/apis/sippy/v1"
 	bqclient "github.com/openshift/sippy/pkg/bigquery"
-	"github.com/openshift/sippy/pkg/componentreadiness/tracker"
```

Comment:
For the changes in this file, metrics and regression tracking are now decoupled.